autobahn/twisted/resource.py

```diff
@@ -34,6 +34,7 @@
     # starting from Twisted 12.2, NoResource has moved
     from twisted.web.resource import NoResource
 from twisted.web.resource import IResource, Resource
+from twisted.python.compat import _PY3
```

Comment:
I'll replace it with the six version, the t.p.compat one is muscle memory for me.

Also, please don't use Twisted < 12. Buildbot should be testing against newer versions -- 12 hasn't been supported for at least three years.